### DISCUSSION TITLE
您好我想请教一下这些您是怎么整理出来的

### DISCUSSION THREAD

**Discussion by yuanjize** [golang-design/history#19] [General]
您好我想请教一下这些您是怎么整理出来的，从第一个commit一个个看吗。比如说我issue上没有特别明确的标签。

**Comment by changkun** [comment]
你好，这个仓库里面的内容确实是手工整理出来的。最早是因为编写 https://github.com/golang-design/under-the-hood 而开始慢慢整理得来。
